scripts/pcm-dpc/covid-19/preprocess.py

```diff
@@ -0,0 +1,193 @@
+        TEMPLATE = ('Node: E:pcm-dpc->E{index}\n'
+                    'typeOf: dcs:StatVarObservation\n'
+                    'variableMeasured: dcs:{SVname}\n'
+                    'observationAbout: {geoNode}\n'
+                    'observationDate: C:pcm-dpc->Date\n'
+                    'value: C:pcm-dpc->{SVname}\n\n')
+        idx = 1
+        with open(self.name + '.tmcf', 'a') as f_out:
+            for statvar in self.stat_vars:
+                f_out.write(TEMPLATE.format_map({'index': idx,
+                    'SVname': statvar, 'geoNode': geo_node}))
+                idx += 1
+
+    def _translate(self):
+        """translate coloumn names from italian to english"""
+        it2en = {'data': 'Date', 'stato': 'State',
+                 'codice_regione':'RegionCode',
+                 'denominazione_regione': 'RegionName',
+                 'codice_provincia': 'ProvinceCode',
+                 'denominazione_provincia': 'ProvinceName',
+                 'sigla_provincia': 'ProvinceAbbreviation',
+                 'lat': 'Latitude', 'long': 'Longitude',
+                 'ricoverati_con_sintomi': 'Count_MedicalConditionIncident'
+                 '_COVID_19_PatientHospitalizedWithSymptoms',
+                 'terapia_intensiva': 'Count_MedicalConditionIncident'
+                 '_COVID_19_PatientInICU',
+                 'totale_ospedalizzati': 'Count_MedicalConditionIncident'
+                 '_COVID_19_PatientHospitalized',
+                 'isolamento_domiciliare': 'Count_MedicalConditionIncident'
+                 '_COVID_19_PatientInHomeIsolation',
+                 'totale_positivi': 'Count_MedicalConditionIncident'
+                 '_COVID_19_ActiveCase',
+                 'variazione_totale_positivi': 'IncrementalCount_Medical'
+                 'ConditionIncident_COVID_19_PositiveCase',
+                 'nuovi_positivi': 'IncrementalCount_MedicalConditionIncident'
+                 '_COVID_19_ActiveCase',
+                 'dimessi_guariti': 'CumulativeCount_MedicalConditionIncident'
+                 '_COVID_19_PatientRecovered',
+                 'deceduti': 'CumulativeCount_MedicalConditionIncident_COVID_19'
+                 '_PatientDeceased',
+                 'totale_casi':'CumulativeCount_MedicalConditionIncident_'
+                 'COVID_19_PositiveCase',
+                 'tamponi': 'CumulativeCount_MedicalTest_COVID_19',
+                 'casi_testati': 'CumulativeCount_Person_COVID_19_Tested',
+                 'casi_da_sospetto_diagnostico': 'PositiveCasesFromClinicActivity',
+                 'casi_da_screening': 'PositiveCasesFromSurveyAndTest'}
+        for col in self.data.columns:
+            assert col in it2en
+        self.data = self.data.rename(columns=it2en)
+
+    def set_location(self):
+        raise NotImplementedError
+
+    def geo_template(self):
+        raise NotImplementedError
+
+STAT_VARS = ['Count_MedicalConditionIncident_COVID_19_'
+             'PatientHospitalizedWithSymptoms',
```

Comment:
```suggestion
STAT_VARS = [('Count_MedicalConditionIncident_COVID_19_'
             'PatientHospitalizedWithSymptoms'),
```
Using parentheses + using yapf will help format these so it is easier for a human to see when line continuations are present.